detectron2/structures/masks.py

```diff
@@ -343,3 +349,24 @@ def crop_and_resize(self, boxes: torch.Tensor, mask_size: int) -> torch.Tensor:
         if len(results) == 0:
             return torch.empty(0, mask_size, mask_size, dtype=torch.bool, device=device)
         return torch.stack(results, dim=0).to(device=device)
+
+    def area(self):
+        """
+        Computes area of the mask.
+        Only works with Polygons, using the shoelace formula:
+        https://stackoverflow.com/questions/24467972/calculate-area-of-polygon-given-x-y-coordinates
+
+        Returns:
+            Tensor: a vector, area for each instance
+        """
+
+        area = []
+        for polygons_per_instance in self.polygons:
+            area_per_instance = 0
+            for p in polygons_per_instance:
+                area_per_instance += polygon_area(p[0::2], p[1::2])
+            area.append(area_per_instance)
+
+        # TODO: rewrite the helper function in torch when the PolygonMasks are
+        # rewritten to use torch
```

Comment:
this TODO needs to be removed since it's unlikely it will be written with torch

Reply:
yeah but why not?

Reply:
removed TODO in 021ec57 as requested